### PR TITLE
DC-6588 Scala 3 upgrade

### PR DESCRIPTION
**Pull request by kjainsc**

### CODE REVIEW

**Comment by kjainsc** [comment]
Rebuild

**Comment by kjainsc** [comment]
Rebuild

File: app/uk/gov/hmrc/contactadvisors/views/Home.scala.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 @this(layout: Layout)
 
-@(string: String)(implicit request: Request[_], messages: Messages)
+@(string: String)(implicit request: Request[?], messages: Messages)
```
Comment (ardwalker) [inline app/uk/gov/hmrc/contactadvisors/views/Home.scala.html:19]: Does this work?

**Review by ardwalker — APPROVED** [review]

**Review by satyajosyula2020 — APPROVED** [review]

File: app/uk/gov/hmrc/contactadvisors/views/Home.scala.html (hunk 1 above)
Comment (kjainsc) [reply app/uk/gov/hmrc/contactadvisors/views/Home.scala.html:19]: @ardwalker - that was one of the warning due to change in wildcard args - https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html

**Pull request merged**